### DISCUSSION TITLE
Interest check: Subsonic API server

### DISCUSSION THREAD

**Discussion by GeorgeHahn** [librespot-org/librespot#1114] [General]
I've built a [Subsonic API](http://www.subsonic.org/pages/api.jsp) server on top of librespot. It allows one to re-stream Spotify songs using any Subsonic client. The real value to me is that it gives access to my whole Spotify library with the missing songs filled in from my personal library.

Is there any interest in developing something like this under the librespot project or within the greater librespot org?

**Comment by kingosticks** [comment]
I personally think that restreaming spotify is a step too far over their terms for this org. If you *must* release it publicly, keep it separate.

**Reply by roderickvd** [reply]
I second this. Retransmitting over a network is explicitly forbidden by their ToS.

**Reply by GeorgeHahn** [reply]
Thanks for the heads up, I hadn't realized this, though it's obvious in retrospect. I'll keep my project private and continue contributing little fixes and enhancements to librespot as I come across them.

**Reply by roderickvd** [reply]
That would be really appreciated, thanks.
